photutils/psf/tests/test_photometry.py

```diff
@@ -580,7 +576,7 @@ def test_psf_photometry_moffat():
     for n in ['x', 'y']:
         assert_allclose(f[n + '_0'], f[n + '_fit'], rtol=1e-3)
     # image was created with a gaussian, so flux won't match exactly
-    assert_allclose(f['flux_0'], f['flux_fit'], rtol=1)
+    assert_allclose(f['flux_0'], f['flux_fit'], rtol=1e-1)
```

Comment:
Why this change?  shouldn't this be unaffected by the rest of the changes in this PR?

Reply:
This is the fix I (perhaps too briefly) mention above, with the improvements to ``assert_allclose``. Previously the incompatibility between the test image creation routine and the fitting routine meant that the test failed, essentially. This meant that, in order to pass the test, the tolerance had to be inflated significantly -- in this case with a tolerance of 100%, or do the input and output flux agree to a factor between 0.5 and 2. As the two functions (PDF Gaussian evaluated at pixel center vs integration over a pixel) weren't _that_ wrong this criterion was passed, but more or less resulted in a test that was wrong, but meaningless; a test with ``rtol=1`` might as well just not have been run in the first place.

I lowered this to 10% so that the test can now actually fail if incompatible changes are made to either routine which affect their results by a non-negligible but not massive amount. Unfortunately I now see that I did not remove the explaining comment for such a high tolerance, which should probably be removed to avoid confusion in the future.